tools/rush-update/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "rush-update",
+  "version": "1.1.1",
+  "description": "Utility to auto update dependencies in all rush projects",
+  "main": "dist/main.js",
+  "bin": {
+    "rush-update": "dist/cli.js"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsc",
+    "lint": "tslint -p ."
```

Comment:
Possible to reuse common tslint like here https://github.com/binaris/shiftjs/blob/ee6e17d3f3ec371ebd9aa0ad2182160cc103bdc2/db/package.json#L12.

Reply:
Changed and also removed all obsolete `tslint.yml` files